### PR TITLE
0.5.core attendee validation

### PR DESCRIPTION
**Pull request by ethanchung936**
- Added validation to the attendee info form.
- Fixed ui so the form scrolls when the screen is too small.
- Moved the header to the attendee activity rather than individual fragments.
- Moved the home button along with the header.

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeActivity.java [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
-import com.budiyev.android.codescanner.CodeScannerView;
-import com.budiyev.android.codescanner.DecodeCallback;
-import com.example.noram.R;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 import com.google.android.material.navigation.NavigationBarView;
-import com.google.zxing.Result;
-
-import org.checkerframework.common.subtyping.qual.Bottom;
 
 public class AttendeeActivity extends AppCompatActivity {
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/AttendeeActivity.java:18]: this isn't necessarily related to your changes but we should add javadoc here

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeProfileFragment.java [hunk 2]
```diff
@@ -87,37 +89,101 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+        }
+    }
+
+    /**
+     * Set the fields of the view to the attendee's information
+     * @param attendee the attendee whose information is being displayed
+     * @param view the view that is being displayed
+     */
+    public void setFields(Attendee attendee, View view) {
+        // Get the necessary fields from the view
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/AttendeeProfileFragment.java:147]: nit: could make these properties of the fragment so we don't have to call findViewById every time we want to set the fields

**Review by ColeDewis — APPROVED** [review]
couple small comments but overall looks good

**Review by gdumouli — APPROVED** [review]
Looks great!
That header bar will be pretty useful

**Pull request merged**